eks/modules/services/scfcharts.tf

```diff
@@ -0,0 +1,29 @@
+# Install UAA using Helm Chart
```

Comment:
We need to be able to support both embedded and separate charts, maybe a separate story. We could add this as a flag for users to indicate whether they want embedded or separate.

Reply:
Agreed. Let's handle it in a separate task.